src/main/java/mods/railcraft/common/blocks/tracks/outfitted/kits/TrackKitDumping.java

```diff
@@ -34,7 +34,7 @@ public TrackKits getTrackKitContainer() {
 
     @Override
     public void onMinecartPass(EntityMinecart cart) {
-        if (!isPowered()) {
+        if (!isPowered() && !theWorldAsserted().getBlockState(getPos().down()).getBlock().isVisuallyOpaque()) {
```

Comment:
That block does not hurt to players or villagers, etc. They cannot suffocate when their fepet is in block